hepcrawl/spiders/elsevier_spider.py

```diff
@@ -554,20 +537,28 @@ def _get_ref_years(self, ref):
     def _parse_references(self, ref, label):
         """Parse all the references."""
         reference = {}
-        textref = ref.xpath(".//ce:textref//text()").extract()
+
+        def add_items_to_reference(**kwargs):
```

Comment:
I was actually thinking of something a bit more generic, at the global level, so you can reuse the same function on all the places...